web/static/js/actions/questionnaire.js

```diff
@@ -115,12 +115,11 @@ export const addStep = (stepType) => {
 
 export const save = () => {
   return (dispatch, getState) => {
-    const questionnaire = getState().questionnaire
-
-    if (questionnaire.id == null) {
-      api.createQuestionnaire(questionnaire.projectId, questionnaire)
-    } else {
+    const questionnaire = getState().questionnaire.data
+    if (questionnaire.id) {
```

Comment:
Why this instead of `questionnaire.id == null`? I'm a bit weary of falsiness in JS